openqa-investigate

```diff
@@ -374,7 +374,17 @@ investigate() {
     comment_id=$(sync_via_investigation_comment "$id" "$first_cluster_job_id") || rc=$?
     [[ $rc != 255 ]] && return $rc
 
-    out=$(trigger_jobs "$id" "${@:2}") || return $?
+    echo "------- comment_id: $comment_id"
+    out=""
+    if ! out=$(trigger_jobs "$id" "${@:2}"); then
+        echo "------- out: $out"
+        client-delete-job-comment "$first_cluster_job_id" "$comment_id"
+        echoerr "Unable to trigger investigation jobs for job $id"
+        return 1
```

Comment:
If there is no comment written how does to cover AC3 from https://progress.opensuse.org/issues/180218#Acceptance-criteria "Given openqa-investigate is started on a job When it aborts (e.g. because the clone failed) Then there is exactly one comment informing that it aborted (and possibly why)"?

Reply:
It could be an improvement to change the comment description, I'm working on it!

Reply:
I saw the suggestion https://progress.opensuse.org/issues/180218#Suggestions but I'm not sure how to incorporate it